index.bs

```diff
@@ -247,6 +247,202 @@ a generator to a processor (e.g., with a call like
 the Web Platform-exposed signals, but ignores any internal custom signals as there is no
 way for the platform sinks to know what the upstream source is.
 
+## MediaStreamTrack ## {#generator-as-track}
+A {{MediaStreamTrackGenerator}} is a {{MediaStreamTrack}}. This section adds
+clarifications on how a {{MediaStreamTrackGenerator}} behaves as a
+{{MediaStreamTrack}}.
+
+### clone ### {#generator-clone}
+The {{MediaStreamTrack/clone}} method on a {{MediaStreamTrackGenerator}}
+returns a new {{MediaStreamTrack}} object whose underlying source is the
```

Comment:
That's what I thought initially. However, returning a generator would actually be confusing since it wouldn't be clear what the writable field in the generator would be. Should it be a new writable (in which case, the generator wouldn't be a clone)? We could make it be the same writable as the original, but I see little value in that.
Returning a MediaStreamTrack removes any confusion and provides the behavior the original clone provides.